drivers/hmis/app/graphql/types/hmis_schema/enums/hud.rb

```diff
@@ -0,0 +1,611 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+# THIS FILE IS GENERATED, DO NOT EDIT DIRECTLY
+
+module Types::HmisSchema::Enums::Hud
```

Comment:
Single file with all the HUD enums, this feels much cleaner to me. I still need to update tests but the schema generation is working fine.